helix-view/src/document.rs

```diff
@@ -104,6 +104,10 @@ pub fn normalize_path(path: &Path) -> PathBuf {
     ret
 }
 
+pub fn canonicalize_path(path: &Path) -> std::io::Result<PathBuf> {
+    std::env::current_dir().map(|current_dir| normalize_path(&current_dir.join(path)))
+}
```

Comment:
Isn't there https://doc.rust-lang.org/std/fs/fn.canonicalize.html?

Reply:
There is but it errors out when the path doesn't exist and we don't want that.

Reply:
Can you please add a comment to explain this for future developers?

Reply:
Good idea, done.